indicate incompatible parameter format (#4926)
centreon/centreon-documentation: cloud/monitoring/basic-objects/hosts.md

* The **Name** field defines the host name that will be used by the Monitoring Engine. The following characters are not authorized: `~!$%^&|'"<>?,()=*{}` and spaces. Using names containing `/` can also have technical impacts on SQL queries, on the interface or cause other compatibility problems with third-party tools. We recommend that you only use `_-.` for host names.
* The **Alias** field shows the alias of the host.
* The **Address** field defines the IP address or DNS name of the host. The **Resolve** button enables us to

>   resolve the domain name by questioning the DNS server configured on the SaaS platform. Note that for security reasons, addresses formated as `<protocol>://<ip-address>` are not accepted.

Reviewer: ```suggestion
  resolve the domain name by questioning the DNS server configured on the SaaS platform. Note that for security reasons, addresses formatted as `<protocol>://<ip-address>` are not accepted.
```

>   resolve the domain name by questioning the DNS server configured on the SaaS platform. Note that for security reasons, parameters formated as `<protocol>://<ip-address>` are not accepted.

Reviewer 2: "parameters" pas sûr de la formulation.
J'aurais plutôt vu un truc du genre "Note that for security reasons, you can't specify address formated as..." 
Not my call really

>   resolve the domain name by questioning the DNS server configured on the SaaS platform. Note that for security reasons, parameters formated as `<protocol>://<ip-address>` are not accepted.

Author: ```suggestion
  resolve the domain name by questioning the DNS server configured on the SaaS platform. Note that for security reasons, addresses formated as `<protocol>://<ip-address>` are not accepted.
```

* The **SNMP Community & Version** fields contain the name of the community and the SNMP version.
* **Custom macros**:

>    * The **Name** and **Value** field are used to define the name and value of the macro. Note that for security reasons, parameters formated as `<protocol>://<ip-address>` are not accepted.

Reviewer: ```suggestion
   * The **Name** and **Value** field are used to define the name and value of the macro. Note that for security reasons, parameters formatted as `<protocol>://<ip-address>` are not accepted.
```

>    * The **Name** and **Value** field enable us to define the name and value of the macro. Note that for security reasons, parameters formated as `<protocol>://<ip-address>` are not accepted.

Reviewer 2: "enable us" ça me parait pas très idiomatique non?
Again, not my call really...

>    * The **Name** and **Value** field enable us to define the name and value of the macro. Note that for security reasons, parameters formated as `<protocol>://<ip-address>` are not accepted.

Author: ```suggestion
   * The **Name** and **Value** field are used to define the name and value of the macro. Note that for security reasons, parameters formated as `<protocol>://<ip-address>` are not accepted.
```
